### PR TITLE
Add python3.9 support

### PR DESCRIPTION
**Pull request by heckad**
https://github.com/bswck/commonvars/issues/1

### CODE REVIEW

File: commonvars/__init__.py [hunk 1]
```diff
@@ -37,13 +37,13 @@ def set(self, value: _T) -> Any:
 
 
 def commonvar_descriptor(
-    cls: type[_T] | None,
+    cls: "type[_T] | None",
```
Comment (bswck) [inline commonvars/__init__.py:40]: I'd be very happy to merge this PR unless it had manual string annotations like here.
Can we just make it `from __future__ import annotations` on top and not touch particular annotations?

**Review by bswck — CHANGES_REQUESTED** [review]
Thanks a lot for contributing! Please have a look at the comment below and I'll merge your PR as quickly as possible. 🚀

**Review by bswck — APPROVED** [review]
Perfect. 💯

**Comment by heckad** [comment]
New release?

**Comment by bswck** [comment]
> New release?

Ready!

**Pull request merged**